manifests/service.pp

```diff
@@ -34,4 +34,12 @@
     status     => 'service pulp_resource_manager status | grep "node resource_manager"',
   }
 
+  service { 'pulp_streamer':
+    ensure     => running,
+    require    => [Service[mongodb]],
```

Comment:
@cristifalcas after some discussion, we do agree that the mongodb service should not be required for this service, its already required for all the other services so we should probably just address them all together.